src/typescript-service.ts

```diff
@@ -325,6 +325,10 @@ export class TypeScriptService {
 				const configuration = this.projectManager.getConfiguration(fileName);
 				// Ensure all files have been added
 				configuration.ensureAllFiles();
+				const program = configuration.getProgram();
+				if (!program) {
+					return Observable.from([]);
```

Comment:
you can just return `[]`, `mergeMap` implicitly calls `Observable.from()` on the return value